src/core/Select/Select.tsx

```diff
@@ -175,7 +172,11 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
 
   useTheme();
 
-  const [isOpen, setIsOpen] = React.useState(false);
+  const [isOpen, setIsOpen] = React.useState(popoverProps?.visible ?? false);
+  React.useEffect(() => {
+    setIsOpen((open) => popoverProps?.visible ?? open);
+  }, [popoverProps]);
```

Comment:
Maybe `popoverProps?.visible` would work in dep list?

Reply:
That's what I had initially, but the unit test was failing 😕 I don't think it was updating